Kickstarter-iOS/ViewModels/SettingsNotificationPickerViewModel.swift

```diff
@@ -0,0 +1,62 @@
+import KsApi
+import Library
+import Prelude
+import ReactiveSwift
+import Result
+
+protocol SettingsNotificationPickerViewModelOutputs {
+  var didTapFrequencyPickerButton: Signal<Void, NoError> { get }
```

Comment:
I think this reads like an input rather than an output. The output should emit _what_ side-effect should happen in the VC, not necessarily the event that triggered it.

So actually in this case it would be something like `notifyDelegateDidTapFrequencyButton`.